packaging/centos/cri-resource-manager.spec.in

```diff
@@ -0,0 +1,33 @@
+Name:    cri-resource-manager
+Version: __VERSION__
+Release: 0
+Summary: A CRI proxy with hardware resource aware workload placement policies.
+License: ASL 2.0 
+URL:     https://github.com/intel/cri-resource-manager
+Source0: https://github.com/intel/cri-resource-manager/archive/cri-resource-manager-%{version}.tar.bz2
+#Source1: cri-resource-manager.service
+#Source2: dockershim.service
+#Source3: 10-override-cri-resource-manager.conf
+BuildRequires: golang >= 1.12
+
+%description
+Kubernetes Container Runtime Interface proxy service with hardware resource aware workload
+placement policies.
+
+%prep
+%setup -q -n cri-resource-manager
+
+%build
+#
+# We opted to always rebuild in our current Makefile. Since we don't want to uselessly
+# build binaries twice, don't bother doing anything here... delay builing to install.
+#
+# make build
+
+%install
+make DESTDIR=%{buildroot} BINDIR=%{_bindir} install
```

Comment:
in almost all distros nowadays there is %make_install macro.

Reply:
Fixed.